crates/cairo-annotations/src/annotations/profiler.rs

```diff
@@ -29,12 +31,46 @@ pub struct ProfilerAnnotationsV1 {
     pub statements_functions: HashMap<StatementIdx, Vec<FunctionName>>,
 }
 
+lazy_static! {
```

Comment:
Rust currently has `std::sync::LazyLock`, which serves the same purpose as `lazy_static!`. This means we can do the following:

```rust
static RE_LOOP_FUNC: LazyLock<Regex> = LazyLock::new(|| {
    Regex::new(r"\[expr\d*\]").expect("Failed to create regex for normalizing loop function names")
});
static RE_MONOMORPHIZATION: LazyLock<Regex> = LazyLock::new(|| {
    Regex::new(r"<.*>")
        .expect("Failed to create regex for normalizing monomorphized generic function names")
});
```

This allows us to remove the `lazy_static` dependency.

Reply:
https://github.com/software-mansion/cairo-annotations/pull/42/commits/8473297369b506c3d9be710352397f2d8ae4cf06